src/storage/v2/storage.hpp

```diff
@@ -324,6 +325,16 @@ class Storage final {
 
     void FinalizeTransaction();
 
+    uint64_t GetTransactionId() const;
+
+    bool IsTransactionActive() const;
+
+    std::atomic<bool> *TransactionAbortedByUser();
+
+    void AbortTransactionByUser();
+
+    bool transaction_aborted_by_user = false;
+
```

Comment:
You can change it to 
```suggestion
bool transaction_aborted_by_user{false};
```

Reply:
You can change it to 
```suggestion
bool transaction_aborted_by_user{false};
```

Reply:
And is there reason why variable is not stored as `std::atomic<bool>` when it is used that way?

Reply:
Are you using this anywhere? If not, you can remove it.

Reply:
Because, storage accessor has access to transaction, and there is no need for this variable

Reply:
yes, yes this is old, needs to be deleted